### ISSUE TITLE
filter on particles appears to be asymmetric 

### ISSUE THREAD

**Issue by PrometheusPi** [ComputationalRadiationPhysics/isaac#86]
When viewing particles (visualized from `0` to `1`) the appearing structure looks asymmetric in x. 

Is it possible that the filter applied to which particles are actually visualized is 
 - depending on just one vector component of the momentum 
 - is depending on the sign of the quantity (and not its absolute value (see issue #85 on why this could not be tested with a suitable functor)
- or both ?

A plot showing this asymmetry is given below
<img width="503" alt="grafik" src="https://user-images.githubusercontent.com/5121158/53483195-4091cd00-3a81-11e9-8ef1-e84bfea90f28.png">

The pink particles show:
 - an ejection front only on the bottom - (due to symmetry reasons they should be in both directions)
 - an asymmetric (higher energy) region inside the laser - (due to the rotational symetry of the laser and its linear polarization this should be approximately mirror-symmetric on the k-B plane) 

This might be a unsuited default setup for particles - but with the missing `length`functor this can not be avoided.

**Comment by ax3l** [comment]
You should post the functor chain you are using as well together with the transfer function settings.

**Comment by PrometheusPi** [comment]
as mentioned - the simulation crashed which causes the client to shut down the simulation viewer and thus the functor chain 😞  - I can not exactly recreate the functor chain.
It was something like
```
length(mul(2))
# or
mul(2) length
```

Either way - it should not crash.

**Comment by ax3l** [comment]
By default, if a "vector" value hits the end of the functor chain it's ".x()" value will be taken.

Functor chains are written as follows:
```
mul(2) | length
```

with a `|` pipe in between.

There is no need for
```
length | mul(x)
```
as the last "stretch" can simply be set in the transfer function.

> I can not exactly recreate the functor chain.

I do not understand what this means. You don't remember what you typed in the textbox?

**Comment by PrometheusPi** [comment]
@ax3l : Yes I did apply the functor chain this morning after sitting the live sim for 12 hours - I do not remember the exact syntax. The green "OK" next to the pipeline suggested a syntax sanity check which allows to experiment and does not cause a total crash of the PIConGPU simulation. 

As already mentioned via mail by @FelixTUD and as written above by @ax3l the correct absolute momentum approach would be:
```
length | mul(2.2) 
```
 @FelixTUD can this be set as default for particle chains?

**Comment by ax3l** [comment]
No, there are little defaults you can set besides using an experimental command-line `tcp` client such as [iltis](https://github.com/ComputationalRadiationPhysics/iltis) or hacking it directly in the code.
